### PR TITLE
116 pathway view options bug fix

### PR DESCRIPTION
**Pull request by lucygilhooly**

### CODE REVIEW

File: src/components/ViewOptions/ViewOptions.jsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import "./ViewOptions.scss";
 import { useState } from "react";
 
-const ViewOptions = () => {
```
Comment (OmarKarame) [inline src/components/ViewOptions/ViewOptions.jsx:4]: Line 4: Please pass in a function as a prop so we can change routes from container.

**Review by OmarKarame — CHANGES_REQUESTED** [review]
Check comments

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import Layout from "../../components/Layout/Layout";
+import ViewOptions from "../../components/ViewOptions/ViewOptions";
```
Comment (OmarKarame) [inline src/pages/Home/Home.jsx:2]: Please remove this import

**Review by OmarKarame — CHANGES_REQUESTED** [review]
Please check comments and don't forget to remove component rendering in layout

**Review by OmarKarame — APPROVED** [review]

**Review by Connz17 — APPROVED** [review]

**Pull request merged**